README.md

```diff
@@ -12,6 +12,7 @@ Blocking access to the internet can help:
 NetGuard is possibly the first free and open source no-root firewall for Android.
 
 Features:
+--------------------------------
```

Comment:
Please remove these, the contents are too small to justify a separate section/link